src/test/java/org/broadinstitute/hellbender/engine/filters/CountingVariantUnitTest.java

```diff
@@ -0,0 +1,308 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.*;
+
+public final class CountingVariantUnitTest {
+
+    // Mirrors CountingVariantFilterUnitTest
```

Comment:
This file *is* the counting variant unit test - I would just remove this comment since it was relevant in the counting read filter tests, but its not here.)